src/components/svgSetting/svgSettingOptions.js

```diff
@@ -1,35 +1,43 @@
-import React, { Component } from 'react';
-import './../checkBoxSelection/checkBoxSelection.scss';
-import {CardLayout} from './svgCardLayout';
-import SvgSetting from './svgSetting';
-import Slider from './slider';
+import React, { Component } from "react";
+import "./../checkBoxSelection/checkBoxSelection.scss";
+import { CardLayout } from "./svgCardLayout";
+import SvgSetting from "./svgSetting";
+import Slider from "./slider";
 class SvgSettingOptions extends Component {
- 	constructor(props) {
-	 	super(props);
-	 	this.state = {
-	 	settingOptionList : []
-        };
-	 }
+  constructor(props) {
+    super(props);
+    this.state = {
+      settingOptionList: []
+    };
+  }
 
-    render() {
-    	for(var key in CardLayout) {
-		  if(CardLayout[key].type==='checkbox'){
-					 this.state.settingOptionList.push(<SvgSetting key={key} option={CardLayout[key].text} />);
-		  	}else if(CardLayout[key].type==='slider'){
-					 this.state.settingOptionList.push(<div className='slider'><Slider key={key} option={CardLayout[key].text} /></div>);
-	 		}
-	 	}
-	 
- 	return (
-	 		<div className='card-settings' >
-	      <div >
-			 		<label> {'SVG settings'} </label>
-		 		</div>
-		 		<div >
-		 			{ this.state.settingOptionList}
-		 		</div>
-		 	</div>
-	 	);
- 	}
-} 
+  render() {
+    for (var key in CardLayout) {
+      if (CardLayout[key].type === "checkbox") {
+        this.state.settingOptionList.push(
+          <SvgSetting
+            key={key}
+            value={CardLayout[key].value}
+            option={CardLayout[key].text}
+          />
+        );
+      } else if (CardLayout[key].type === "slider") {
+        this.state.settingOptionList.push(
```

Comment:
this is not how state is local state should be mutated, please make use of setState.

Reply:
While I was clearing console,  I removed this loop altogether its was causing warnings regarding key duplication. components with the same key value